modules/viewing-stats-collected-kubernetes-nmtate-op.adoc

```diff
@@ -0,0 +1,38 @@
+// This is included in the following assemblies:
+//
+// networking/k8s_nmstate/k8s-nmstate-about-the-k8s-nmstate-operator.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="viewing-stats-collected-kubernetes-nmtate-op_{context}"]
+= Viewing statistics collected by the Kubernetes NMState Operator
+
+The Kubernetes NMState Operator, `kubernetes-nmstate-operator`, can collect node network performance statistics and pass them to the `nmstate-metrics` API. You can enter a command on the OpenShift CLI (`oc)` to view these statistics.
```

Comment:
`node network performance statistics` <-- I think this feature is more about feature usage and not about performance? Or maybe both?